src/payment/strategies/checkoutcom-custom/checkoutcom-apm/checkoutcom-apm-payment-strategy.ts

```diff
@@ -0,0 +1,43 @@
+import { InternalCheckoutSelectors } from '../../../../checkout';
+import { OrderRequestBody } from '../../../../order';
+import { PaymentArgumentInvalidError } from '../../../errors';
+import { PaymentInstrument, WithDocumentInstrument } from '../../../payment';
+import { PaymentRequestOptions } from '../../../payment-request-options';
+import CheckoutcomCustomPaymentStrategy from '../checkoutcom-custom-payment-strategy';
+
+export default class CheckoutcomAPMPaymentStrategy extends CheckoutcomCustomPaymentStrategy {
+
+    protected async _executeWithoutHostedForm(payload: OrderRequestBody, options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { payment, ...order } = payload;
+        const paymentData = payment && payment.paymentData;
+
+        if (!payment || !paymentData) {
+            throw new PaymentArgumentInvalidError(['payment.paymentData']);
+        }
+
+        await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
+
+        const _paymentData = {
```

Comment:
the underscore prefix is a bit unusual, maybe you don't need to declare it here?

```js
this._store.dispatch(this._paymentActionCreator.submitPayment({
  ...payment,
  paymentData: {
    ...paymentData,
    formattedPayload: this._createFormattedPayload(payment.methodId, paymentData),
  }
}));